tapdb/assets_common.go

```diff
@@ -259,8 +258,11 @@ func upsertGroupKey(ctx context.Context, groupKey *asset.GroupKey,
 		return nullID, fmt.Errorf("unable to insert internal key: %w",
 			err)
 	}
+	// Insert the group key with an empty tapscript root since we only
+	// support signatures as asset group key witnesses for now.
 	groupID, err := q.UpsertAssetGroupKey(ctx, AssetGroupKey{
 		TweakedGroupKey: tweakedKeyBytes,
+		TapscriptRoot:   []byte{},
```

Comment:
Perhaps add this onto the struct now so harder to forget to insert it down the line? May save some debugging saga.